examples/gallery/plot/points-categorial.py

```diff
@@ -0,0 +1,56 @@
+"""
+Color points by categories
+---------------------------
+The :meth:`pygmt.Figure.plot` method can be used to plot symbols which are
+color-coded by categories.
+"""
+
+import numpy as np
+import pandas as pd
+import pygmt
+
+# Load sample penguins data and convert 'species' column to categorical dtype
+df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/penguins.csv")
+df["species"] = df.species.astype(dtype="category")
+
+# Use pygmt.info to get region bounds (xmin, xmax, ymin, ymax)
+# The below example will return a numpy array like [2.  4.4 4.3 7.9]
+region = pygmt.info(
+    table=df[["bill_length_mm", "bill_depth_mm"]],  # x and y columns
+    per_column=True,  # report output as a numpy array
+    spacing="3/2",  # rounds x and y intervals by 3 and 2 respectively
```

Comment:
Feel like we should be able to do `spacing = [3, 2]` but not implemented yet in `pygmt.info` :/

Reply:
ok, thought it's working properly